README.md

```diff
@@ -40,8 +40,21 @@ Special thanks to those who have made contributions to Alibaba's internal JDK bu
 
 Technologies included in Alibaba Dragonwell have been published in following papers
 
-* ICSE'19：https://2019.icse-conferences.org/event/icse-2019-technical-papers-safecheck-safety-enhancement-of-java-unsafe-api
+## 2021
 
-* ICPE'18: https://dl.acm.org/citation.cfm?id=3186295
+- Yifei Zhang, Tianxiao Gu, Xiaolin Zheng, Lei Yu, Wei Kuai, Sanhong Li [**Towards a Serverless Java Runtime**](https://ase21-industry.hotcrp.com/doc/ase21-industry-paper7.pdf?cap=07ax_GWBvNW-0U) In _(ASE'21) (To appear)_
+
+## 2020
+
+- Mingyu Wu, Ziming Zhao, Yanfei Yang, Haoyu Li, Haibo Chen, Binyu Zang, Haibing Guan, Sanhong Li, Chuansheng Lu, Tongbao Zhang: [**Platinum: A CPU-Efficient Concurrent Garbage Collector for Tail-Reduction of Interactive Services**](https://www.usenix.org/system/files/atc20-wu-mingyu.pdf) In _USENIX Annual Technical Conference 2020: 159-172 (ATC'20)_
```

Comment:
Change `In _USENIX Annual Technical Conference 2020: 159-172 (ATC'20)_` to `In _USENIX ATC 2020_, pp. 159-172`

Reply:
done